docs/user/openstack/install_upi.md

```diff
@@ -29,6 +29,7 @@ Below is a step-by-step guide to a UPI installation that mimics an automated IPI
 * [Ignition Config](#ignition-config)
   * [Infra ID](#infra-id)
   * [Bootstrap Ignition](#bootstrap-ignition)
+    * [Glance Authentication](#glance_authentication)
```

Comment:
Sorry, one tiny nit, but this link doesn't work when viewed on Github. It should say `#glance-authentication` (a dash instead of an underscore).

Reply:
done! thank you for noticing